student.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
```

Comment:
### Optional
- [ ] It would be nice if we use require_relative for files in our directory